nats/tests/nats_test.rs

```diff
@@ -141,3 +141,59 @@ async fn send_publish(_opt: &TestOptions) -> RpcResult<()> {
 
     Ok(())
 }
+
+// /// test unlinking during request loop
```

Comment:
Left over debug?

Reply:
:astonished: Sorry I don't know how I left this in! the test was originally here before our discussion, will remove :bow: